template/koa-rest-api/package.json

```diff
@@ -1,8 +1,13 @@
 {
   "dependencies": {
     "@koa/router": "^10.1.1",
+    "@opentelemetry/api": "^1.0.3",
+    "@opentelemetry/exporter-collector-grpc": "0.25.0",
+    "@opentelemetry/instrumentation-aws-sdk": "0.1.0",
+    "@opentelemetry/instrumentation-http": "0.26.0",
+    "@opentelemetry/sdk-node": "0.26.0",
     "@seek/logger": "^5.0.1",
-    "skuba-dive": "^1.2.0",
+    "aws-sdk": "^2.1039.0",
```

Comment:
question: do you need to declare this as a dep?

Reply:
I think it's up for discussion. It's needed for `aws-sdk` instrumentation, which I guess is highly desirable for anyone at seek. Perhaps some don't use `aws-sdk` but I bet it's the minority.

Reply:
I think when I flipped through the GitHub page it said you could use aws-sdk V3 as well?

Reply:
Actually, otel won't work with aws-sdk < V3. At least it wasn't in early testing. This was because in V2 of the aws SDK, request signing was happening _before_ otel header propagation. This meant that all instrumented requests to AWS services via the AWS SDK would fail on account of invalid signatures. This wasn't an issue in V3

Reply:
@bernos re `aws-sdk` v2 - I can confirm instrumentation works for it with current otel libraries.

Reply:
Would it be better to just leave it out and let consumers byo sdk?

Reply:
It depends on your definition of "better", as always :). Do we want to bring in `aws-sdk` instrumentation for consumers out of the box? 

<img width="724" alt="Screen Shot 2021-12-01 at 4 21 38 pm" src="https://user-images.githubusercontent.com/903687/144176796-27d04f76-837d-4ada-b9e9-281bc3bd469e.png">

Looking at this trace - I see clear advantages of knowing what the HTTP POST was actually is about, which in the case with `aws-sdk` instrumentation is very illustrative. With minimal built-in `http` instrumentation it'll be just another opaque HTTP request.

The problem is https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-sdk needs aws-sdk (or rather its types)